library/core/src/cmp.rs

```diff
@@ -730,13 +734,12 @@ impl<T: Clone> Clone for Reverse<T> {
 /// This trait can be used with `#[derive]`.
 ///
 /// When `derive`d on structs, it will produce a
-/// [lexicographic](https://en.wikipedia.org/wiki/Lexicographic_order) ordering
-/// based on the top-to-bottom declaration order of the struct's members.
+/// [lexicographic](https://en.wikipedia.org/wiki/Lexicographic_order) ordering based on the
+/// top-to-bottom declaration order of the struct's members.
 ///
-/// When `derive`d on enums, variants are ordered primarily by their discriminants.
-/// Secondarily, they are ordered by their fields.
-/// By default, the discriminant is smallest for variants at the top, and
-/// largest for variants at the bottom. Here's an example:
+/// When `derive`d on enums, variants are ordered primarily by their discriminants. Secondarily,
+/// they are ordered by their fields. By default, the discriminant is smallest for variants at the
+/// top, and largest for variants at the bottom. Here's an example:
```

Comment:
I realize this is just a whitespace change and not really your fault, but I thought I should mention this is a natural consequence of [implicit discriminants](https://doc.rust-lang.org/reference/items/enumerations.html#implicit-discriminants) and there are *much* more confusing cases than this documentation suggests which nonetheless one might assume would be determined by some "default" behavior, see [this Playground example](https://play.rust-lang.org/?version=stable&mode=debug&edition=2021&gist=e8af409e88637534cb154dd3c2a96072).

Reply:
While I think I understand what you mean, it's not clear to me what I should do with that information. Do you want me to reword the section?

Reply:
No need in particular, tbh.